isis/src/control/objs/BundleConstraint/BundleConstraint.cpp

```diff
@@ -0,0 +1,18 @@
+#include "BundleConstraint.h"
+
+namespace Isis {
+
+  /**
+   * Constructor
+   *
+   */
+  BundleConstraint::BundleConstraint() {
+  }
+
+
+  /**
+   * Destructor
+   */
+  BundleConstraint::~BundleConstraint() {
+  }
+}
```

Comment:
I am confused by the purpose of this object.

Reply:
Originally, there was a second constraint sub-class that handled continuity between polynomial segments but now there isn't. I thin Ken had hoped that this would eventually be an interface for a generic constraint in the bundle adjust but there's not actually an interface here. I'm going to take it out.